### ISSUE TITLE
remember to include The center loss parameters in the optimizer, othewise it will not work

### ISSUE THREAD

**Issue by mohamedr002** [KaiyangZhou/pytorch-center-loss#22]

**Comment by toannguyen1904** [comment]
can you give us an example?

**Comment by cotyyang** [comment]
> can you give us an example?

maybe he mean is this. in main.py 80 line `optimizer_centloss = torch.optim.SGD(criterion_cent.parameters(), lr=args.lr_cent)`

**Comment by cotyyang** [comment]
> can you give us an example?

maybe he mean is this. in main.py 80 line `optimizer_centloss = torch.optim.SGD(criterion_cent.parameters(), lr=args.lr_cent)`

**Comment by mohamedr002** [comment]
> > can you give us an example?
> 
> maybe he mean is this. in main.py 80 line `optimizer_centloss = torch.optim.SGD(criterion_cent.parameters(), lr=args.lr_cent)`

Yes, exactly, this is what I meant, the center loss parameters are trainable parameters and should be updated during training, by excluding these parameter from the optimizer, it will not change,

**Comment by fafafafafafafa** [comment]
why i put  optimizer_centloss into optimizer, raise ValueError("optimizer got an empty parameter list"?
anyone can help me?
